scripts/install/jdownloader.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# JDownloader Installer for swizzin
+# Author: Aethaeran
+
+# References
+# https://swizzin.ltd/dev/structure/
+# https://www.digitalocean.com/community/tutorials/how-to-install-java-with-apt-on-debian-10
+# https://support.jdownloader.org/Knowledgebase/Article/View/install-jdownloader-on-nas-and-embedded-devices
+# Liara already made a doc for installing JDownloader manually https://docs.swizzin.net/guides/jdownloader/
+# https://linuxize.com/post/how-to-check-if-string-contains-substring-in-bash/
+# https://linuxize.com/post/bash-check-if-file-exists/
+
+# TODO: Figure out if these imports are even necessary
+. /etc/swizzin/sources/globals.sh
+. /etc/swizzin/sources/functions/users
```

Comment:
Not necessary, these get pulled in whenever `box` is ran and are exported into the shell for the whole `box` run

Reply:
I had a feeling if I left a TODO about that someone would chime in for me. Appreciated!